src/js/player.js

```diff
@@ -3891,6 +3891,22 @@ TECH_EVENTS_RETRIGGER.forEach(function(event) {
  * @type {EventTarget~Event}
  */
 
+/**
+ * Apply plugin to player.
+ *
+ * @method Player#usePlugin
+ * @param   {string} name
+ *          The name under witch the plugin will be saved. Under this same
+ *          name `player.usingPlugin(name)`. Must be a string and must not
```

Comment:
`Must be a string`, is this true? It cannot be a symbol nor number? Not that I care for numbers, but I would be for symbols in future. Also, storing plugin in `WeakMap`, if available, might be preferable.

Reply:
You can't replicate symbol, therefor they would be a safe option for internal plugins. Other than that, no... It is just a primitive for this kind of things..

Reply:
You can't replicate symbol, therefor they would be a safe option for internal plugins. Other than that, no... It is just a primitive for this kind of things.

Reply:
You can't replicate symbol, therefor they would be a safe option for internal plugins. Other than that, no... It is just a primitive for this kind of things.